partitura/utils/music.py

```diff
@@ -941,14 +960,22 @@ def estimate_symbolic_duration(dur, div, eps=10**-3):
     if np.abs(qdur - DURS[i]) < eps:
         return SYM_DURS[i].copy()
     else:
-        # NOTE: Guess tuplets (Naive) it doesn't cover composite durations from tied notes.
-        type = SYM_DURS[i + 3]["type"]
-        normal_notes = 2
-        return {
-            "type": type,
-            "actual_notes": math.ceil(normal_notes / qdur),
-            "normal_notes": normal_notes,
-        }
+        # Note when the duration is not found, the we are left with two solutions:
+        # 1. The duration is a tuplet
+        # 2. The duration is a composite duration
+        # For composite duration. We can use the following approach:
+        j = find_nearest(COMPOSITE_DURS, qdur)
+        if np.abs(qdur - COMPOSITE_DURS[j]) < eps and return_com_durations:
+            return copy.copy(SYM_COMPOSITE_DURS[j])
+        else:
+            # NOTE: Guess tuplets (Naive) it doesn't cover composite durations from tied notes.
+            type = SYM_DURS[i + 3]["type"]
+            normal_notes = 2
+            return {
+                "type": type,
+                "actual_notes": math.ceil(normal_notes / qdur),
+                "normal_notes": normal_notes,
+            }
```

Comment:
does this function ever return None as specified in the function documentation?

Reply:
Good catch, I have to update the documentation.

Reply:
Addressed with commit [61881a48](https://github.com/CPJKU/partitura/pull/352/commits/61881a482f6cc70fd2184f8a6546f22c0afd2172)